packages/pwa-kit-runtime/package.json

```diff
@@ -62,6 +62,7 @@
     "sinon": "^13.0.1",
     "superagent": "^6.1.0",
     "supertest": "^4.0.2",
+    "uuid": "^9.0.0",
```

Comment:
I see you are using the latest here 👍 ... might want to make sure we are using the lasted in all projects as mentioned earlier.